lib/tasks/add_navigation_links.rake

```diff
@@ -22,48 +22,93 @@ namespace :navigation_links do
   heart_icon = twemoji_path("heart.svg")
   rainbowdev = image_path("rainbowdev.svg")
 
+  def perform_create_of_navigation_links?
+    # Someone really wants this
+    return true if ApplicationConfig["CREATE_NAVIGATION_LINKS"]
+
+    # This logic echoes the InternalPolicy behavior which is used in the
+    # Admin::APpliciationController.
+    return false if User.with_any_role(*Authorizer::RoleBasedQueries::ANY_ADMIN_ROLES).any?
+
+    # May as well provide a way for them to skip
+    return false if ApplicationConfig["SKIP_CREATE_NAVIGATION_LINKS"]
```

Comment:
Are we sure about that? I think some things assume these links will be there, so maybe YAGNI until we tackle this more comprehesively?